src/jsonapi.js

```diff
@@ -139,7 +139,22 @@ export const readEndpoint = (endpoint, {
         .then(json => {
           dispatch(apiRead({ endpoint, ...json }));
           onSuccess(json);
-          resolve(json);
+
+          const prevUrl = getPaginationUrl(json, 'prev', apiHost, apiPath);
+          const nextUrl = getPaginationUrl(json, 'next', apiHost, apiPath);
+
+          const getPrevPage = () => dispatch(readEndpoint(prevUrl));
+          const getNextPage = () => dispatch(readEndpoint(nextUrl));
+
+          const resObj = {
```

Comment:
I don't think that's helpful enough to the client. We can do better than that imo.

I was imagining having a class which would be the argument that we're passing through the fulfillment handler. Could look something like this:

```js 
class ApiResponse {
  constructor(prevUrl, nextUrl) {
    this.prevUrl = prevUrl;
    this.nextUrl = nextUrl;
  }

  loadPrev() {
    dispatch(readEndpoint(this.prevUrl));
  }

  loadNext() {
    dispatch(readEndpoint(this.nextUrl));
  }
}
```